app/build.gradle

```diff
@@ -58,10 +58,10 @@ dependencies {
     api libraries.androidArchLifecycleExtensions
     api libraries.constraintLayout
     api libraries.dagger
+    api libraries.glide
```

Comment:
I swapped Picasso out for Glide because of this - https://github.com/ataulm/art-collector/pull/19/commits/e92412daebff1c510bf1307a0085097af09af53c - I didn't want to spend more time on this issue since it's not the aim of this PR/the rest of the project.

Reply:
I created an issue for this https://github.com/ataulm/art-collector/issues/20